docs/endpoints/endpoints.rst

```diff
@@ -0,0 +1,737 @@
+Globus Compute Endpoints
+************************
+
+A Globus Compute Endpoint is an agent (i.e., a service; a process) launched by the user
+to serve as a conduit for executing functions on a computing resource.  The Compute
+Endpoint agent (CEA) manages the site‑specific interactions for executing functions,
```

Comment:
> I think we should reserve the term "agent" when specifically referring to the installable package. In most cases, including this one, we refer to the process running on the machine, which we can simply call the Compute Endpoint.

That ... took more wordsmithing than I had thought it would.

> I also recommend avoiding acronyms like CEA. A user that, for example, jumps directly to the Getting Started section will not know what CEA stands for.

Yeah, I've been waffling on this as well.  I'm not so concerned about jumping around -- just search for it and learn it -- but the larger point for me is that CEA is just awkward as an acronym.  Moreover, we're not strictly in the academic realm so probably safe to just elide it completely.

Reply:
Yea a new acronym would be confusing if it's not widely known, unless its definition is above uses on the same page, to reduce lengthy redundancy.  I don't think it was repeated too many times here?  So 👍 to hide it (except for internal links that you are using it for)